### ISSUE TITLE
My own profile page is 500'ing

### ISSUE THREAD

**Issue by wizjo** [railsbridge/bridge_troll#308]
https://www.bridgetroll.org/users/4207/profile returns 500 :(
![screen shot 2015-03-04 at 7 58 12 pm](https://cloud.githubusercontent.com/assets/167792/6499041/204844f6-c2a9-11e4-88d6-bae1dc793130.png)

**Comment by tjgrathwell** [comment]
Hmm. Right now you (and four other lucky users) don't have a `Profile` row associated with your `User` in the database.

I can fix it in the console, but I'm not sure how it ended up like that. Were you ever able to see your profile page before? Did anything unusual happen during your registration, if you can recall?

**Comment by wizjo** [comment]
Not sure how exactly this happened. I did go back to editing my profile right after registration, bumped into some form errors requesting current password but eventually form was submitted.

**Comment by tjgrathwell** [comment]
Okay, I wasn't really able to get to the bottom of why these profiles have disappeared but I fixed up the profiles for the five users who were missing them. Any skills/bio information will need to be re-entered, if desired.

I've also added another validation on user in c23a2e96ae37dc212dfc007bbf8c631ecd6a0778 so that hopefully a user can never again be saved without a `Profile` associated.

Thanks for bringing this up! Our exception tracker seems to have stopped sending emails so I'll fix that up sometime soon too.
